app/src/main/java/fr/free/nrw/commons/logging/LogsSender.java

```diff
@@ -20,164 +20,170 @@
 import java.util.zip.ZipOutputStream;
 
 import fr.free.nrw.commons.auth.SessionManager;
+import fr.free.nrw.commons.utils.BetaCheckerUtil;
 import timber.log.Timber;
 
 /**
  * Abstract class that implements Acra's log sender
  */
 public abstract class LogsSender implements ReportSender {
```

Comment:
why does this class have so many changes?

Reply:
Sorry, I must've messed something with the formatter. The change I made was just to remove isBeta parameter from the constructor and its usages in the file to directly use to Util function.

Reply:
It would be great if you could undo it. :)

Reply:
Made the change. Is this alright now?

Reply:
Great! Can you also rename the `BetaCheckerUtil` class?

Reply:
Should we go with `BuildConfigUtils` as @sup95  suggested? Or do you want me to use some other name?

Reply:
Does `ConfigConstantsUtils` seem better?

Reply:
`ConfigUtils` looks better to me or maybe we can simply put the method in `Utils` which already exists

Reply:
It's done from my end :)